content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -358,13 +362,11 @@ var tracer = new Tracer(settings);
 Tracer.Instance = tracer;
 ```
 
-**Note:** Settings must be set on `TracerSettings` _before_ creating the `Tracer`. Changes made to `TracerSettings` properies after the `Tracer` is created are ignored.
-
 {{% /tab %}}
 
 {{% tab "JSON file" %}}
 
-To configure the Tracer using a JSON file, create `datadog.json` in the instrumented application's directory. The root JSON object must be a hash with a key/value pair for each setting. For example:
+To configure the Tracer using a JSON file, create `datadog.json` in the instrumented application's directory. The root JSON object must be hashed with a key/value pair for each setting. For example:
```

Comment:
"A hash" is used colloquially in JSON to refer to objects (because they behave like hash tables). Let's change "a hash" to "an object."

```suggestion
To configure the Tracer using a JSON file, create `datadog.json` in the instrumented application's directory. The root JSON object must be an object with a key/value pair for each setting. For example:
```